### ISSUE TITLE
Validations "QOR Support" section

### ISSUE THREAD

**Issue by bodhi** [qor/doc#10]
Seems unnecessary? 

> QOR is architected from the ground up to accelerate development and deployment of [...]

Part of it describes the way it integrates with QOR admin, this is probably useful to keep, but just edit it to remove any info that is already described.

**Comment by raven-chen** [comment]
Removed unnecessary QOR support text.

**Comment by coreform** [comment]
Those are remnants of the per-repo README documentation. For the new monolithic Doc, I have been changing that section to 'QOR Integration', removing the basic QOR introduction paragraph and replacing with something like:

`Although Location could be used alone, it integrates nicely with QOR.`

E.g.:
https://github.com/qor/doc/blob/master/plugins/i18n.md
https://github.com/qor/doc/blob/master/plugins/l10n.md

https://github.com/qor/doc/commit/adccebfe6a1b7253c5923a46ac53d57130903b75
^ shows some work to improve that section. It is not redundant as it makes it clear that such Plugins can be used standalone. Ideally, we would be achieving clarity _and_ consistency, rather than just consistency as a result of this Issue.

**Comment by coreform** [comment]
Reopening as it would be ideal to follow the new pattern for `QOR Integration` sections (consistency) in applicable Plugins.

**Comment by bodhi** [comment]
> Although Location could be used alone, it integrates nicely with QOR

This seems odd: this is the _QOR_ doc, and Location is being described as a QOR plugin, so it would be strange if it _didn't_ integrate nicely?

**Comment by raven-chen** [comment]
Yeah. Better to separate it into two sections. "Integrate with QOR" and "Use it alone". Let's just focus on the first section for now ?

**Comment by coreform** [comment]
It would be nice if there were more information about _how_ each Plugin integrates with QOR (because that is effectively _what_ QOR is: an integration of components and Plugins). Like the information provided in https://github.com/qor/doc/blob/master/plugins/activity.md , that would give developers a better understanding of what to expect to see and the behaviour in QOR Admin when a Plugin is used.

**Comment by raven-chen** [comment]
@coreform yes. https://doc.getqor.com/helpers/render.html this is a good negative example. I'm working on it now.
